packages/notivue/core/animations.css

```
@@ -1,27 +1,15 @@
-
-[data-notivue-align='bottom'] {
-   & .Notivue__enter,
-   & .Notivue__leave {
-      --notivue-ty: 200%;
-   }
-}
-
-.Notivue__enter {
-   animation: Notivue__enter-kf 350ms cubic-bezier(0.5, 1, 0.25, 1);
+:root {
+   --nv-enter-animation: Notivue__enter-kf 350ms cubic-bezier(0.5, 1, 0.25, 1);
```

Stylelint (17.12.0)

[error] Expected "Notivue__enter-kf" to be "notivue__enter-kf" (value-keyword-case)

(value-keyword-case)